deploy/example_catalog/cr-app-kafka55.json

```diff
@@ -0,0 +1,215 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "confluentkafka55"
+    },
+    "spec" : {
+        "systemdRequired" : true,
+        "defaultPersistDirs" : ["/var/log", "/opt/configscripts"],
+        "capabilities" : [
+            "SYS_PACCT",
+            "SYS_RESOURCE",
+            "CHOWN",
+            "FSETID",
+            "KILL",
+            "SETGID",
+            "SETUID",
+            "NET_BIND_SERVICE",
+            "NET_BROADCAST",
+            "SYS_PTRACE",
+            "SETFCAP"
+        ],
+
+    "config": {
+        "roleServices": [
+            {
+                "serviceIDs": [
+                    "control-center"
+                ],
+                "roleID": "control-center"
+            },
+            {
+                "serviceIDs": [
+                    "zookeeper"
+                ],
+                "roleID": "zookeeper"
+            },
+            {
+                "serviceIDs": [
+                    "broker"
+                ],
+                "roleID": "broker"
+            },
+            {
+                "serviceIDs": [
+                    "schema-registry"
+                ],
+                "roleID": "schema-registry"
+            },
+            {
+                "serviceIDs": [
+                    "rest-proxy"
+                ],
+                "roleID": "rest-proxy"
+            },
+            {
+                "serviceIDs": [
+                    "kafka-connect"
+                ],
+                "roleID": "kafka-connect"
+            },
+            {
+                "serviceIDs": [
+                    "ksqldb"
+                ],
+                "roleID": "ksqldb"
+            },
+            {
+                "serviceIDs": [
+                    "kafka-client"
+                ],
+                "roleID": "kafka-client"
+            }
+        ],
+        "selectedRoles": [
+            "control-center",
+            "zookeeper",
+            "broker",
+            "schema-registry",
+            "rest-proxy",
+            "kafka-connect",
+            "ksqldb",
+            "kafka-client"
+        ]
+    },
+    "defaultImageRepoTag": "docker.io/bluedata/kafka:1.0",
```

Comment:
I have done the changes, but since my updates are already merged, it is added as a new PR. Please review the same : Kubedirector conf kafka 5.5 app yaml updates #332